static/styles/zulip.css

```diff
@@ -1419,6 +1425,10 @@ div.focused_table {
     display: block;
     position: relative;
     overflow: hidden;
+    max-height: 1000vh;
```

Comment:
I did this to allow expand and collapse animation. 
My aim was to set value as unlimited. 

Is there any better approach or alternative to it?

Reply:
I'm not sure, but I'd at least add a clear comment.  To be clear, is that an alternative to `unset` or something?

Reply:
Yeah, I tried different various values like `unset`, `100%`, `max-content` but none of them worked.
I have added the comment!